tests/testthat/test-mlaps.R

```diff
@@ -0,0 +1,171 @@
+# Helper function to create dummy lab data needed for mLAPS unit tests
+dummy_lab <- function(id, omop, value, unit, mintime){
+  res <- data.table(
+    genc_id = rep(id, length(value)),
+    test_type_mapped_omop = omop,
+    result_value = value,
+    result_unit = rep(unit, length(value)),
+    collection_date_time =  format(as.POSIXct(mintime, tz = "UTC") + sample(0:(24*60*60 - 1), size=length(value), replace = TRUE),"%Y-%m-%d %H:%M")
+  )
+  return(res)
+}
+
+# Helper function to create dummy admdad data needed for mLAPS unit tests
+dummy_admdad <- function(id, admtime){
+  res <- data.table(
+    genc_id = id,
+    admission_date_time = format(as.POSIXct(admtime, tz = "UTC"),"%Y-%m-%d %H:%M")
+  )
+  return(res)
+}
+
+####### test 1
```

Comment:
Note that all tests currently fail because there is no input argument called `hours_offset` (should be called `hours_after_admission`).

Reply:
Thanks for catching this!

Reply:
Great, thanks - test are all running fine now.